### ISSUE TITLE
[Problem] Check + Redumper = Error

### ISSUE THREAD

**Issue by mnadareski** [SabreTools/MPF#454]
Using MPF.Check with Redumper currently does the absolutely wrong thing. It doesn't parse anything and it says the dumping program was `dd` as a result. Fix this please thanks.

**Comment by mnadareski** [comment]
TL;DR on investigation:
- The only way for `dd` to be selected from everything I can tell is for you to pass in "dd" as the dumping program. The default dumping program is DIC.
- mict's test from yesterday used a combination of the short flag (`-u`) and an all-lower-case "redumper". Hiccup's original one used `-u` and sentence case "Redumper". acrid's used `--use` and sentence case "Redumper"
- The above indicates it should be an issue with capitalization... if I didn't already have `ToLowerInvariant` in the code to handle exactly that

So, somehow, `--use Redumper` is being interpreted as `--use dd`

**Comment by acridAxid** [comment]
I'm using the current stable build from GitHub.

`mpf.check cdrom audio --use Redumper <my-image>.cue`

Here's the full output, which looks like it made no attempt to parse the image at all:
[!submissionInfo.txt](https://github.com/SabreTools/MPF/files/10829248/submissionInfo.txt)

**Comment by acridAxid** [comment]
Output from [the now-current build from AppVeyor](https://ci.appveyor.com/project/mnadareski/mpf/build/2.4-1031/artifacts):
[!submissionInfo.txt](https://github.com/SabreTools/MPF/files/10829279/submissionInfo.txt)

The stable build creates a `.zip` file named after the image file; when `--use Aaru` is passed, this `.zip` file contains the `!submissionInfo.txt` as well as all the Aaru-generated logging, and when `--use Redumper` is passed, the `.zip` file contains nothing but the bungled `!submissionInfo.txt` from above.

The current build only creates the `!submissionInfo.txt` and does not generate the `.zip` file.  Not sure if that's an intended artifact of the testing build.

**Comment by mnadareski** [comment]
The zip was enabled by default erroneously with last stable. The lack of zip is intentional unless you use `-z`/`--zip`. That is incidental to the issue at hand.

**Comment by mnadareski** [comment]
I have a feeling that those who were having issues all were using a stable build. Redumper support was not completed as of the last stable and may have included placeholder info. This would explain why `dd` is showing up for those users. I'm closing this because it's not an actual issue since the problem has been fixed in WIP builds.
